data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -196,7 +198,7 @@ podman run \
 	--network=host \
 	--entrypoint=/bootkube \
 	"{{.BootkubeImage}}" \
-	start --asset-dir=/assets --required-pods openshift-kube-apiserver/openshift-kube-apiserver,kube-system/kube-scheduler,openshift-kube-controller-manager/openshift-kube-controller-manager,openshift-cluster-version/cluster-version-operator
```

Comment:
the scheduler should not be dropped here, just the name must be adapted to phase 3.